template/.csscomb.json

```diff
@@ -0,0 +1,376 @@
+      "font-family",
+      "font-size",
+      "font-weight",
+      "font-style",
+      "font-variant",
+      "font-size-adjust",
+      "font-stretch",
+      "font-effect",
+      "font-emphasize",
+      "font-emphasize-position",
+      "font-emphasize-style",
+      "font-smooth",
+      "line-height"
+    ],
+    [
+      "position",
+      "z-index",
+      "top",
+      "right",
+      "bottom",
+      "left"
+    ],
+    [
+      "display",
+      "visibility",
+      "float",
+      "clear",
+      "overflow",
+      "overflow-x",
+      "overflow-y",
+      "-ms-overflow-x",
+      "-ms-overflow-y",
+      "clip",
+      "zoom",
+      "-webkit-align-content",
+      "-ms-flex-line-pack",
+      "align-content",
+      "-webkit-box-align",
+      "-moz-box-align",
+      "-webkit-align-items",
+      "align-items",
+      "-ms-flex-align",
+      "-webkit-align-self",
+      "-ms-flex-item-align",
+      "-ms-grid-row-align",
+      "align-self",
+      "-webkit-box-flex",
+      "-webkit-flex",
+      "-moz-box-flex",
+      "-ms-flex",
+      "flex",
+      "-webkit-flex-flow",
+      "-ms-flex-flow",
+      "flex-flow",
+      "-webkit-flex-basis",
+      "-ms-flex-preferred-size",
+      "flex-basis",
+      "-webkit-box-orient",
+      "-webkit-box-direction",
+      "-webkit-flex-direction",
+      "-moz-box-orient",
+      "-moz-box-direction",
+      "-ms-flex-direction",
+      "flex-direction",
+      "-webkit-flex-grow",
+      "-ms-flex-positive",
+      "flex-grow",
+      "-webkit-flex-shrink",
+      "-ms-flex-negative",
+      "flex-shrink",
+      "-webkit-flex-wrap",
+      "-ms-flex-wrap",
+      "flex-wrap",
+      "-webkit-box-pack",
+      "-moz-box-pack",
+      "-ms-flex-pack",
+      "-webkit-justify-content",
+      "justify-content",
+      "-webkit-box-ordinal-group",
+      "-webkit-order",
+      "-moz-box-ordinal-group",
+      "-ms-flex-order",
+      "order"
+    ],
+    [
+      "-webkit-box-sizing",
+      "-moz-box-sizing",
+      "box-sizing",
+      "width",
+      "min-width",
+      "max-width",
+      "height",
+      "min-height",
+      "max-height",
+      "margin",
+      "margin-top",
+      "margin-right",
+      "margin-bottom",
+      "margin-left",
+      "padding",
+      "padding-top",
+      "padding-right",
+      "padding-bottom",
+      "padding-left"
+    ],
+    [
+      "table-layout",
+      "empty-cells",
+      "caption-side",
+      "border-spacing",
+      "border-collapse",
+      "list-style",
+      "list-style-position",
+      "list-style-type",
+      "list-style-image"
+    ],
+    [
+      "content",
+      "quotes",
+      "counter-reset",
+      "counter-increment",
+      "resize",
+      "cursor",
+      "-webkit-user-select",
+      "-moz-user-select",
+      "-ms-user-select",
+      "user-select",
+      "nav-index",
+      "nav-up",
+      "nav-right",
+      "nav-down",
+      "nav-left",
+      "-webkit-transition",
+      "-moz-transition",
```

Comment:
These prefix defs probably not needed. There should never be any vendor prefix other than in extreme circumstances.

Reply:
The prefixes inclusion in the sort-order settings has no impact on if someone does or does not write them in their stylesheets. This only makes sure that "if they do" they are sorted correctly. If you don't want someone using the prefixes, then that is better handled by a code review or possibly in a lint rule if that is supported.